WordPress/gutenberg
# [RNMobile] Correct color palette in color settings

## Description

Fixes:https://github.com/wordpress-mobile/gutenberg-mobile/issues/2222

ref to gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2224

Correct color palette in color settings.

## How has this been tested?

1. Open mobile app
2. Add Buttons block
3. Expect to see not duplicated colors 

## Screenshots 

<img width="300" alt="Screenshot 2020-05-06 at 14 27 53" src="https://user-images.githubusercontent.com/22746080/81176718-d68a8700-8fa5-11ea-86f9-4df5c2cd5092.png">

## Types of changes

* Removed duplicated colors
* Use `uniq` to avoid duplicates
* Correct active segment to be dependent on `currentValue`

## Checklist:
